lib/DDG/Spice/Holiday.pm

```diff
@@ -13,81 +13,80 @@ spice wrap_jsonp_callback => 0;
 spice from => '([^/]+)/([^/]+)/([^/]+)/([^/]+)';
 spice to => 'http://www.timeanddate.com/scripts/ddg.php?m=whenis&c=$1&q=$2&y=$3&callback={{callback}}';
 
-my @triggers  = share('holidays.txt')->slurp(chomp => 1);
-my $countries = join('|', share('countries.txt')->slurp(chomp => 1));
+my @triggers  = sort { length $b <=> length $a } share('holidays.txt')->slurp(chomp => 1);
 
 triggers any => @triggers;
 
+my $triggers_re = join('|', @triggers);
```

Comment:
Please can we stick to the coding convention within this file, which is to use camel case for variable name